### DISCUSSION TITLE
Group queue enabled only on weekends (Friday/Saturday/Sunday)

### DISCUSSION THREAD

**Discussion by Contrubius** [chromiecraft/chromiecraft#4443] [Suggestions]
PvP Battlegrounds on workdays (95% WSG) are low populated, especially in the EU morning hours, therefore any premade that joins the queue greatly unbalances the play and makes counterplays almost impossible for the random opposing team. By that, Battlegrounds mostly become 3v4, 4v5, or even 4v7 or greater in favor of one team. Disabling the group queue would balance up and even the odds. This is much less noticeable on the weekends when Call to Arms is available, because most battlegrounds are full.

### Alternative suggestion: 
Trying to implement a waiting queue in the algorithm, for example, game is currently 4v4, one player cannot enter the BG until another player queues on the opposing team. Same for the premade, premade of 3 cannot enter in an even match, until 3 players queue in the opposing team.

**Comment by balleny** [comment]
Something that might be easier to implement overall from a technical point of view:
Group queue only possible on CTA for the CTA bg.

**Reply by Contrubius** [reply]
Every weekend is CtA so I thought that goes without saying.

**Reply by balleny** [reply]
yes, hence the "for the CTA bg" only.
Thus, not allowed group queue for each bg every weekend, but restricted to the CTA specific bg
